documentation/API_documentation/DeviceLocation_API.md

```diff
@@ -0,0 +1,101 @@
+# Overview
+The Location API provides programmable interface for developers and other users to verify 
+if given user equipment is within the range(radial distance) from some place defined by coordinates(longitude, latitude) on the Earth.
+## 1\. Introduction
+Logistics, gaming and many more industry sectors define a need of getting user real location. Moreover, some of them 
+need this functionality to prevent frauds - to compare GPS location with real user equipment location.
+<img src="./resources/Location_overview.png" alt="LOC_VER" title="Location verification">
+
+## 2\. Quick Start
+Location API v0.1.0 exposes only one endpoint, which can be used to determine if user equipment is within the given radial distance from
+given location. To do so we need to pass following parameters in request body: 
+1. **ueId** - an object with four fields, each of them make possible to pass ueId in different format: externalId [1], msisdn, ipv4Addr and ipv6Addr 
+2. **latitude** - a floating point number representing latitude of location, value from -90 to 90
+3. **longitude** - a floating point number representing longitude of location, value from -180 to 180
+4. **accuracy** - an integer representing accuracy (radius) expected for location verification in km
+
```

Comment:
Have introduced another parameter to the list, pls update the documentation accordingly.

https://github.com/camaraproject/DeviceLocation/pull/8

Reply:
First, we have to approve the PR